doc/extension.rst

```diff
@@ -31,7 +34,10 @@ For example:
 Configuration
 -------------
 
-The extension has a few configuration options that can be set in ``conf.py``:
+The extension has a few configuration options that can be set in ``conf.py``.
+
+See also additional configuration options in the :ref:`built-in extensions
+<built-in-extensions>`.
```

Comment:
I'm really nitpicking here, but this feels inferior to the admonition style example I made. Particularly, it's less direct in stating that there is a specific extension which is temporarily enabled by default and thus its configuration variables are a non-negotiable addition to these ones. I think it's that extension that we need a reference for here, not the whole set.

Than again... meh.